app/presenters/formats/local_transaction_presenter.rb

```diff
@@ -7,36 +7,76 @@ def schema_name
     end
 
     def details
+      required_details
+      .merge(optional_details)
+      .merge(external_related_links: external_related_links)
+    end
+
+    def registers_exact_route?
+      false
+    end
+
+  private
+
+    def required_details
       {
         lgsl_code: edition.lgsl_code,
-        lgil_override: edition.lgil_override,
         service_tiers: service_tiers,
+      }
+    end
+
+    def optional_details
+      {}.merge(lgil_override)
```

Comment:
`{}.merge(lgil_override)` is identical to `lgil_override` - so you could save a few characters. Might not look so stylish though